waspc/cli/src/Wasp/Cli/Command/Db/Reset.hs

```diff
@@ -0,0 +1,37 @@
+module Wasp.Cli.Command.Db.Reset
+  ( reset,
+  )
+where
+
+import Control.Concurrent (newChan)
+import Control.Concurrent.Async (concurrently)
+import Control.Monad.IO.Class (liftIO)
+import StrongPath ((</>))
+import System.Exit (ExitCode (..))
+import Wasp.Cli.Command (Command)
+import Wasp.Cli.Command.Common (findWaspProjectRootDirFromCwd)
+import Wasp.Cli.Command.Message (cliSendMessageC)
+import qualified Wasp.Cli.Common as Common
+import qualified Wasp.Generator.DbGenerator.Jobs as DB.Jobs
+import Wasp.Generator.Job.IO (readJobMessagesAndPrintThemPrefixed)
+import qualified Wasp.Message as Msg
+
+reset :: Command ()
```

Comment:
When I ran `db reset` with PG running outside of Docker (like I normally do), I go the following error on WSL2:

```sh
🐝 --- Reseting the database... ---------------------------------------------------


Db: Environment variables loaded from .env
Db: Prisma schema loaded from ../db/schema.prisma
Db: Datasource "db": PostgreSQL database "todoapp", schema "public" at "host:5432"
Db: 
✔ Are you sure you want to reset your database? All data will be lost. … yes
Db: 
Db: Applying migration `20220413175924_switch_to_postgresql`
Db: Applying migration `20220818151104_rename_email_to_username`
Db: Applying migration `20220822154342_add_social_login`
Db: 
Db: Database reset successful
Db: 
Db: The following migration(s) have been applied:
Db: 
Db: migrations/
Db:   └─ 20220413175924_switch_to_postgresql/
Db:     └─ migration.sql
Db:   └─ 20220818151104_rename_email_to_username/
Db:     └─ migration.sql
Db:   └─ 20220822154342_add_social_login/
Db:     └─ migration.sql
Db: 
Db: Running generate... (Use --skip-generate to skip the generators)
Db(stderr): Error: Attempted to load provider value using `env(PRISMA_CLIENT_OUTPUT_DIR)` but it was not present. Please ensure that PRISMA_CLIENT_OUTPUT_DIR is present in your Environment Variables
✅ --- Database reset failed with exit code 1 -------------------------------------
```

Reply:
Ah sorry about this, I still have to test `reset` properly, it is on my TODO list!
This will be interesting, it wants to re-generate clients. I think what I will want to do here is use the `--skip-generate` flag that they mention and then either not do anything, or run our own generation of clients -> I have to think about what is the correct approach.

Reply:
I just added the --skip-generate flag, that should do it, since we take care of generating clients anyway before running any db commands.